api/models/user.py

```diff
@@ -69,6 +70,7 @@ class UserCapacity(Base):
     start = Column("init_date", Date, nullable=False)
     end = Column("end_date", Date, nullable=True)
     user_id = Column("usrid", Integer, ForeignKey("usr.id"), nullable=False)
+    aggregates = Column("aggregates", JSONB, nullable=True)
```

Comment:
Can we call it something else? Aggregates seems too generic and I'm having a hard time to understand what it will store

Reply:
Of course. It will hold the amount of vacation hours the person will have as well as the amount of total expected hours for each year in the time period. 
![Screenshot from 2024-01-08 07-47-33](https://github.com/Igalia/phpreport/assets/3666105/0d0126e2-225d-4658-b5a7-6c2f838bf736)
I had a hard time naming it (obviously ;)). If you don't mind being verbose, we could do `yearly_expected_and_vacation`or just `expected_and_vacation`. Or I am completely open to suggestions.

Reply:
I don't mind verbose in this case, `yearly_expected_and_vacation` works for me, thank you!